SU2_PY/FSI/FSIInterface.py

```diff
@@ -1544,26 +1553,26 @@ def setFluidInterfaceVarCoord(self, FluidSolver):
             if GlobalIndex in self.FluidHaloNodeList[myid].keys():
               posX0, posY0, posZ0 = self.haloNodesPositionsInit[GlobalIndex]
               DispX, DispY, DispZ = self.haloNodesDisplacements[GlobalIndex]
-              if posY0 == 0.0:
-                posX = posX0
-                posY = posY0
-                posZ = posZ0
-              else:
-                posX = posX0 + DispX
-                posY = posY0 + DispY
-                posZ = posZ0 + DispZ
+              #if posY0 == 0.0:
+              #  posX = posX0
+              #  posY = posY0
+              #  posZ = posZ0
+              #else:
+              posX = posX0 + DispX
+              posY = posY0 + DispY
+              posZ = posZ0 + DispZ
               FluidSolver.SetVertexCoordX(self.fluidInterfaceIdentifier, iVertex, posX)
               FluidSolver.SetVertexCoordY(self.fluidInterfaceIdentifier, iVertex, posY)
               FluidSolver.SetVertexCoordZ(self.fluidInterfaceIdentifier, iVertex, posZ)
             else:
-              if self.localFluidInterface_array_Y_init[localIndex] == 0.0:				# !!! This is temporary and case dependent, it should be removed ASAP !!!
-                posX = self.localFluidInterface_array_X_init[localIndex]
-                posY = self.localFluidInterface_array_Y_init[localIndex]
-                posZ = self.localFluidInterface_array_Z_init[localIndex]
-              else:
-                posX = self.localFluidInterface_array_DispX[localIndex] + self.localFluidInterface_array_X_init[localIndex]
-                posY = self.localFluidInterface_array_DispY[localIndex] + self.localFluidInterface_array_Y_init[localIndex]
-                posZ = self.localFluidInterface_array_DispZ[localIndex] + self.localFluidInterface_array_Z_init[localIndex]
+              #if self.localFluidInterface_array_Y_init[localIndex] == 0.0:				# !!! This is temporary and case dependent, it should be removed ASAP !!!

```

Comment:
And these?

Reply:
I did some clean up here. Originally these old FSI scripts should have been removed, but then I realized that some people actually wanted to use it or were already using it like this. So, at the end, I just keep them until the coupling with our environment can be officially achieved.